pom.xml

```diff
@@ -179,6 +179,12 @@
             <artifactId>jboss-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>org.glassfish</groupId>
+          <artifactId>javax.json</artifactId>
+          <version>1.0.4</version>
```

Comment:
Version is best specified in dependencyManagement. This makes the prod build easier.